### PR TITLE
Use friendly names for Ids

### PR DESCRIPTION
**Pull request by dneto0**

### CODE REVIEW

**Comment by dneto0** [comment]
Example output:
vert

File: tools/cfg/bin_to_dot.cpp [hunk 1]
```diff
@@ -41,7 +42,8 @@ const char* kContinueStyle = "style=dotted";
 // a SPIR-V module.
 class DotConverter {
  public:
-  DotConverter(std::iostream* out) : out_(*out) {}
+  DotConverter(libspirv::NameMapper name_mapper, std::iostream* out)
```
Comment (antiagainst) [inline tools/cfg/bin_to_dot.cpp]: Double copy here? Either use `const libspirv::NameMapper&` or use `name_mapper_(std::move(name_mapper))`?

Comment (dneto0) [inline tools/cfg/bin_to_dot.cpp]: I didn't care too much about efficiency.  In this case the name mapper is a lambda that captures a single "this" pointer.  But fixed in  892239b7cc anyway.

**Comment by antiagainst** [comment]
+@

**Pull request merged**